woof-code/rootfs-skeleton/usr/bin/xsaneshell

```diff
@@ -1,6 +1,13 @@
 #!/bin/sh
 #8.06.11 rodin.s: i18n
 
+xsane -v >/dev/null 2>/dev/null
+if [ $? -eq 127 ]; then
+  /usr/lib/gtkdialog/box_splash -bg red -fg white -timeout 5 -text "ERROR: xsane is not installed. Install xsane first."
```

Comment:
maybe add '-box close' else a spurious mouse over may close the message before read.